networkit/graphio.pyx

```diff
@@ -259,7 +259,9 @@ cdef class EdgeListReader(GraphReader):
 	and can be accessed using getNodeMap().
 
 	To shift continuous integer node labels which are not zero-indexed, set firstNode to
-	the smallest id used in the file.
+	the smallest id used in the file. As a value for firstNode different from 0 is not 
+    allowed in the non-continuous case firstNode will be set to 0 regardless of input if
+    continuous is false.
```

Comment:
I would instead write that the value of `firstNode` will be ignored in the non-continuous case, as, again, it is an implementation detail that we set it to 0 instead of not using it.